airbyte-cdk/bulk/core/load/build.gradle

```diff
@@ -20,6 +20,7 @@ dependencies {
 
     testFixturesApi testFixtures(project(':airbyte-cdk:bulk:core:bulk-cdk-core-base'))
 
+    testImplementation("io.mockk:mockk:1.13.12")
```

Comment:
Do we not have a version catalog in airbyte/airbyte? If so, we should probably add one.

Reply:
Unfortunatelly not and I wonder why. This could be added in another PR